html/modules/custom/ncms_ui/src/Entity/Content/ContentBase.php

```diff
@@ -416,4 +417,34 @@ public function getCommonTaxonomiesService() {
     return \Drupal::service('ncms_tags.common_taxonomies');
   }
 
+  /**
+   * Builds the render array for displaying the current results as a table.
+   */
+  public function getTags() {
```

Comment:
Please move this function to appear directly after the `hasTags()` method in the same file.